src/main/resources/db/migration/V42_6__Create_user_enum.sql

```diff
@@ -0,0 +1,8 @@
+do
+$$
+    begin
+        if not exists(select from pg_type where typname = 'specialization_field') then
+            create type "specialization_field" as enum ('COMMON_CORE', 'EL', 'TN');
+        end if;
+    end
+$$;
```

Comment:
"create user enum" is not correct for this content.

Try "create_user_specialization_field_enum for example